public/js/components/RightSidebar.js

```diff
@@ -80,6 +80,11 @@ const RightSidebar = React.createClass({
 
   setupKeyboardShortcuts() {
     const { keyShortcuts } = this.props;
+    if (this.keyShortcutsEnabled) {
+      return;
+    }
+
+    this.keyShortcutsEnabled = true;
```

Comment:
I think context will likely be the better solution for `keyShortcuts`. checking the existence of keyShortcuts would allow many handlers to be bound.

I'm going to merge this as this does not seem like a blocker, but an improvement. Happy to update in a separate issue.